templates.go

```diff
@@ -1107,6 +1120,7 @@ coreos:
       --logtostderr=true \
       --machine-id-file=/rootfs/etc/machine-id \
       --cadvisor-port=4194 \
+      --cloud-provider={{.Cluster.Kubernetes.CloudProvider}} \
```

Comment:
This won't work. The templating is for the Kubernetes block in clustertpr and it doesn't have a CloudProvider.